freqtrade/arguments.py

```diff
@@ -444,6 +444,7 @@ def common_scripts_options(self, subparser: argparse.ArgumentParser = None) -> N
             '-p', '--pairs',
             help='Show profits for only this pairs. Pairs are comma-separated.',
             dest='pairs',
+            required=True
```

Comment:
`--pairs` should not be required if we are going to fetch the whitelist from the config...

Reply:
good point - plot_dataframe did not support loading from whitelist previously, but i've added this now.